main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
```

Comment:
main_test? - but I could also see how that's overkill for this repo at this stage